sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/ISymbolExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2020 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace SonarAnalyzer.Helpers
+{
+    internal static class ISymbolExtensions
+    {
+        public static bool HasAttribute(this ISymbol symbol, KnownType type) =>
```

Comment:
I think it would be better to keep this file in this PR (since this is where we want to get) and do a separate PR in which we replace all the occurrences. I will do this PR today so we will not forget about this.

Reply:
Ok, let's keep that as is then :+1: